### PR TITLE
[+] Added Cloudflare Rocket Loader HTML Injection

### PR DESCRIPTION
**Pull request by j3ssie**
### Template / PR Information

- References: https://developers.cloudflare.com/speed/optimization/content/rocket-loader/enable/

### Template Validation

I've validated this template locally?
- [x] YES
- [ ] NO

### CODE REVIEW

**Comment by j3ssie** [comment]
looking good from my side @DhiyaneshGeek, waiting for it to be merge

**Review by ehsandeep — APPROVED** [review]

**Comment by j3ssie** [comment]
Hi @DhiyaneshGeek, sorry for the late response but you can disable the rocket loader or add a CSP header to fix this

- https://developers.cloudflare.com/speed/optimization/content/rocket-loader/enable/
- https://developers.cloudflare.com/fundamentals/reference/policies-compliances/content-security-policies/#product-requirements

File: http/misconfiguration/cloudflare-rocketloader-htmli.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - https://developers.cloudflare.com/speed/optimization/content/rocket-loader/enable/
+  metadata:
+    max-request: 1
+    verified: true
+  tags: misconfig,cloudflare,htmli
+
+http:
+  - method: GET
+    path:
+      - "{{BaseURL}}/cdn-cgi/image/width=1000,format=auto/https://raw.githubusercontent.com/simple-icons/simple-icons/develop/icons/cloudflare.svg"
```
Comment (smcgivern) [inline http/misconfiguration/cloudflare-rocketloader-htmli.yaml:19]: This does not detect Rocket Loader. This detects the image transformation feature: https://developers.cloudflare.com/fundamentals/reference/cdn-cgi-endpoint/

https://developers.cloudflare.com/images/transform-images#svg-files documents how Cloudflare handles SVG files.

**Comment by Sab0tag3d** [comment]
True, it's detect "image optimization" (from any origin) feature, and this presumably enables "/cdn-cgi/" functionality, which may be similar behavior to Rocket Loader feature.
And this feature are not allow to upload html, or JavaScript execute code

**Pull request merged**